Test/Utils/ComputeNode/Installation.ps1

```diff
@@ -62,6 +62,34 @@ function Uninstall-Agent {
     Invoke-MsiExec -Uninstall -Session $Session -Path "C:\Artifacts\agent\contrail-vrouter-agent.msi"
 }
 
+function Import-ExtensionCertificate {
+    Param ([Parameter(Mandatory = $true)] [Testbed] $Testbed)
+
+    Write-Log 'Importing vRouter certificate to certstore'
+    Invoke-Command -Session $Testbed.GetSession() {
+        $Cert = Import-Certificate -CertStoreLocation Cert:\LocalMachine\Root\ 'C:\Artifacts\vrouter\vRouter.cer'
+        try {
+            Import-Certificate -CertStoreLocation Cert:\LocalMachine\TrustedPublisher\ 'C:\Artifacts\vrouter\vRouter.cer'
```

Comment:
why is one in `try` block but the other one is not? please add comment to the code about this

Reply:
Isn't the catch block explaining why?

Reply:
Yes, that is the reason. Importing twice will not break anything, but the idea behind our tests is: cleanup after yourself if it is possible, leave the place as you found it. That's why all steps that we run during initialization should be atomic in a sense, that they either success or fail, but leave environment untouched. If the function is not working like this, it has CleanupStack as a parameter, and after every step it adds cleanup step.